### DISCUSSION TITLE
Great job!

### DISCUSSION THREAD

**Discussion by AcroTiger** [GAM-team/got-your-back#386] [General]
I rarely post anything anywhere. But...

I wanted to take a quick second to say, "Thank you!"

The ability to backup and restore emails from one Gmail account to another _including labels!!!_ is a feature that you would think exists natively in Gmail, but no. Without this project, I would have been stuck. So, just...

THANK YOU!

**Comment by dab15** [comment]
I would like to add my thanks to the contributors to / authors of this project. You saved my bacon when my alumni association pulled the plug on free storage.
